[ci-visibility] Add git metadata upload section to junit (#18149)
DataDog/documentation: content/en/continuous_integration/tests/junit_upload.md

: Commit committer date in ISO 8601 format.<br/>
**Example**: `2021-03-12T16:00:28Z`

## Git metadata upload

> From `datadog-ci>=2.9.0`, CI Visibility automatically uploads git metadata information (commit history). This metadata contains file names but no file contents. If you want to opt out of this behavior, you can do so by passing the flag `--skip-git-metadata-upload`.

Reviewer: ```suggestion
From `datadog-ci` version `2.9.0` or later, CI Visibility automatically uploads Git metadata information (commit history). This metadata contains file names but no file contents. If you want to opt out of this behavior, pass the flag `--skip-git-metadata-upload`.
```
